core/services/fluxmonitor/flux_monitor.go

```diff
@@ -480,7 +480,7 @@ func (p *PollingDeviationChecker) consume() {
 	defer p.pollTicker.Stop()
 
 	if !p.idleThreshold.IsInstant() {
-		p.idleTicker = time.After(p.idleThreshold.Duration())
+		p.idleTimer = time.After(p.idleThreshold.Duration())
```

Comment:
@spooktheducks when does this get called... it doesn't appear to be in response to any log?

Reply:
I understand this now, its only called once on the first invocation. Need to add a comment here explaining.